static/app/views/starfish/queries/useIndexedSpans.tsx

```diff
@@ -46,7 +59,7 @@ function getEventView(filters: Filters, location: Location, sorts?: Sort[]) {
     {
       name: '',
       query: search.formatString(),
-      fields: Object.values(SpanIndexedField),
+      fields: fields ?? Object.values(SpanIndexedField),
```

Comment:
IMO throw this default out, too! The hook should be as thin and unopinionated in data as possible

Reply:
Removed!